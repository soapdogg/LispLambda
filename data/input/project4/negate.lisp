(DEFUN NEGATE (X)
    (TIMES X (MINUS 0 1))
)

(NEGATE (MINUS 0 13))
(NEGATE 0)
(NEGATE 234)