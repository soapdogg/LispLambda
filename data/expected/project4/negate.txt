13
0
-234